Checkpoint1/Checkpoint1.cs

```diff
@@ -1,12 +1,116 @@
 ﻿using System;
+using System.Linq;
 
 namespace Checkpoint1
 {
-    class Program
+  class Program
+  {
+
+    static void Main(string[] args)
     {
-        static void Main(string[] args)
+      Divisible(" ");
+      Sum(" ");
+      Factor(" ");
+      Random(" ");
+      MinMax(" ");
+
+    }
+
+
+    //This function divides 100 by user input to see how many numbers the chosen number divides into evenly.
+    static int Divisible(string divide) {
+     Console.WriteLine("Enter a number to divide by: ");
+     int divisor = Convert.ToInt32(Console.ReadLine());
+     int count = 0;
+      for (int i = 0; i < 101; i++)
+      {
+        if (i % divisor == 0)
+        {
+          Console.WriteLine(i);
+          count++;
+        }
+      }
+      Console.WriteLine("The count of numbers divisible by " +divisor+ " is: " + count);
+     return divisor;
+   }
+  //This function takes in numbers from a user and returns the sum.
+   static int Sum(string stringAnswer) {
+      string answer = " ";
+      int sum = 0;
+      while (answer != "ok")
```

Comment:
answer is not really checked in your while condition, since you are manually existing the loop using `break` keyword.